### ISSUE TITLE
_type always returns normal account type

### ISSUE THREAD

**Issue by grian32** [Chasesc/OSRS-API-Wrapper#7]
`<AccountType.NORMAL: '/m=hiscore_oldschool/index_lite.ws'>`
^ is what returns even when im calling the hiscores of an ironman or an uim
if this isnt implemented properly yet, please ignore and close this issue

**Comment by Chasesc** [comment]
The RS hiscores do not give the account type, so it is currently expected that the correct account type is passed in when making a hiscores object:

```python
iron_mammal = Hiscores('iron mammal', AccountType.IRONMAN) # account type is passed in explictly
```

This could be updated to add a new default AccountType of AUTO to the Hiscores constructor that will automatically try each type. If you feel this would be useful, feel free to open a PR. If not, I could get around to it whenever I have some time.

**Comment by Chasesc** [comment]
On a second thought, instead of a new AccountType, it would be better to default the type to ```None``` and handle this case in the Hiscores:

```python
class Hiscores(object):
    def __init__(self, username, type=None): # automatically determine type by checking each URL if type is None
```

**Comment by grian32** [comment]
if you could implement that, would be much appreciated, dont really have enough experience and or knowledge to make a pr sadly.

**Comment by Chasesc** [comment]
This should be included as a part of 0.0.3
